crates/ef-testing-proc-macro/src/converter.rs

```diff
@@ -0,0 +1,139 @@
+use std::collections::BTreeMap;
+
+use rayon::prelude::*;
+use serde_json::Value;
+
+use crate::{
+    constants::{FORK, REGEX},
+    content_reader::ContentReader,
+    dir_reader::{DirReader, PathWrapper},
+};
+
+/// The TestConverter is used to convert all the
+/// tests files in to rust tests.
```

Comment:
I don't understand what this means, can this be more specific?

Reply:
I plan to update to this, is this ok?
```
/// The TestConverter is used to convert the directory structure
/// into a String containing all the rust tests to be ran.
///
/// # Example
///
/// Test location: BlockchainTests/GeneralStateTests/stRandom/
/// List of tests: [randomStatetest0.json, randomStatetest1.json, ...]
/// Inner tests: [randomStatetest0_d0g0v0_Shanghai, randomStatetest0_d1g0v0_Shanghai,
/// ..., randomStatetest1_d0g0v0_Shanghai, randomStatetest1_d1g0v0_Shanghai, ...]
/// Generated String:
/// r#"
/// mod randomStatetest0 {
///   use super::*;
///   #[test]
///   fn test_randomStatetest0_d0g0v0_Shanghai() {
///     ...
///   }
///   #[test]
///   fn test_randomStatetest0_d1g0v0_Shanghai() {
///     ...
///   }
///   #[test]
///   fn test_randomStatetest1_d0g0v0_Shanghai() {
///     ...
///   }
///   #[test]
///   fn test_randomStatetest1_d1g0v0_Shanghai() {
///     ...
///   }
///   ...
/// }
/// "#
```

Reply:
This is great